The Long Dark Save Editor 2/Game data/GlobalData.cs

```diff
@@ -312,6 +315,8 @@ public static GearItemSaveDataProxy Create()
             while (MainWindow.Instance.CurrentSave.Global.Inventory.Items.Any(i => i.Gear.m_InstanceIDProxy == id))
                 id = r.Next();
             item.m_InstanceIDProxy = id;
+            item.m_NonInteractive = false;
```

Comment:
These should be unecessary as boolean defaults to false.

Reply:
Just old habit.